tests/hindsight/run/input/test_heka_messages.lua

```diff
@@ -55,7 +55,7 @@ local messages = {
             build = "20170518000419",
             channel = "release",
             docType = "main",
-            size = 4242
+            size = 4242.1
```

Comment:
Isn't this already a double?

Reply:
I suppose it is, how did the tests pass?

Reply:
It doesn't fail because it doesn't run (the schema name is incorrect in the input) it should be `EnvVersion = "heka.telemetry_payload_size.1"`

Reply:
Looking at the original code:
1. why can't we just fix the message to store size as an int64 (does it negatively impact down stream consumers?)
1. even better: the analysis plugin generating these messages is a pretty bad hack requiring a custom cfg to override sensible defaults.  With a small change to the direct to parquet code we can get rid if it all together and stop doubling the message volume going through Hindsight.

Reply:
1. is what mreid proposed earlier. I don't think it will impact downstream consumers since the only ones I am aware of are CEP filters.